src/actions/authActions.js

```diff
@@ -177,7 +179,7 @@ export const loginAction = (
           dispatch(setActiveBlockchainNetworkAction(BLOCKCHAIN_NETWORK_TYPES.ETHEREUM));
         }
       } else {
-        api.init();
+        api.init(null, undefined, null);
```

Comment:
looks very scary

Reply:
Yeah, this is a bit shady. I've added a new parameter (coming on a following PR) which made flow go nuts. I'll try removing this from here and see how it goes. Will report back :+1:

Reply:
Alright, it's not required. Removed.